Source/WebCore/svg/animation/SVGSMILElement.h

```diff
@@ -137,7 +137,7 @@ class SVGSMILElement : public SVGElement {
     enum BeginOrEnd { Begin, End };
     SMILTime findInstanceTime(BeginOrEnd, SMILTime minimumTime, bool equalsMinimumOK) const;
     void resolveFirstInterval();
-    void resolveNextInterval();
+    bool resolveNextInterval();
```

Comment:
I'd like to see a comment saying what the return value means (or make it an enum).

Reply:
I have few follow-up blink merges planned, so I think those might change this entirely. :-)

Reply:
Since I have future plans to merge few more Blink patches in SMIL to make WebKit better. I am going to take it as 'RESOLVED' for time being.